### DISCUSSION TITLE
Shortcut, How to set directory and run command

### DISCUSSION THREAD

**Discussion by ewwink** [microsoft/terminal#16743] [Q&A] [answered]
How to convert the following command to shortcut or batch script

```
wt.exe cmd.exe
cd D:\Apps\pythonEnv
Scripts\activate
```

**Comment by Eyobs-droid** [comment]
Open a text editor and write the following commands:
@echo off
start et.exe
start cmd.exe
cd /d D:\Apps\pythonEnv
call Scripts\activate
then save the file with a .bat extension to convert it into a batch script

**Comment by zadjii-msft** [comment]
Or just, 

```
wt -d D:\Apps\pythonEnv cmd.exe /k Scripts\activate
```

**Comment by ewwink** [comment, marked as answer]
thanks for the response but both of your answer is not working, but it already solved few days ago [13426](https://github.com/microsoft/terminal/issues/13426#issuecomment-1962320301) we need to put the command inside quote
